### PR TITLE
ADDED:Mail Validation and style modification in all pages.

### PR DESCRIPTION
**Pull request by sivabalan18102001**

### CODE REVIEW

File: Sivabalan/15-02-2023 Task/User Edit Page.html
Comment (NavinAT) [inline Sivabalan/15-02-2023 Task/User Edit Page.html:1]: 1. Validation error not showing
2. remove button enlarge style.
3. Change button border radius - refer button style in bootstrap [https://getbootstrap.com/docs/4.0/components/buttons/](url)

**Review by NavinAT — CHANGES_REQUESTED** [review]
I have reviewed your changes and added some comment..

Thanks.

File: Sivabalan/15-02-2023 Task/Detail page Script.js [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+        }
+            else
+            {
+              var numberofuser = document.getElementById("numberofuser").value;
+              var lengthoftable = arr.length;
+              if (numberofuser)
+              {
+                var valid = lengthoftable>=numberofuser;
+                Boolean(valid);
+                if(0<numberofuser && valid)
```
[the remark this answers is not in the document]
Comment (sivabalan18102001) [reply Sivabalan/15-02-2023 Task/Detail page Script.js:28]: completed

File: Sivabalan/15-02-2023 Task/Detail page Script.js [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+    document.write('<tr><td>'+j+'</td><td>'+ user.fname+'</td><td>'+user.mname+'</td><td>'+user.lname+'</td><td>'+user.Court_Name+'</td><td>'+user.Registration_date+'</td><td>'
+                    +user.Gender+'</td><td>'+user.Mobile_number+'</td><td>'+user.Address+'</td><td>'+user.Email+'</td></tr>');
+}
+function editUser()
+{
+    var obj = localStorage.getItem("testJSON");
+    var arr = JSON.parse(obj);
+    if(obj)
+    {
+        if(arr[0]==null)
```
[the remark this answers is not in the document]
Comment (sivabalan18102001) [reply Sivabalan/15-02-2023 Task/Detail page Script.js:16]: completed

File: Sivabalan/15-02-2023 Task/Main page Script.js [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+    var mailformat = /^\w+([\.-]?\w+)*@\w+([\.-]?\w+)*(\.\w{2,3})+$/;
+        if(email.value.match(mailformat))
+        {
+            var Email = document.getElementById("email").value;
+        }
+        else
+        {
+            alert("Invalid email address!");
+            var Email = "";                        
+        }
```
[the remark this answers is not in the document]
Comment (sivabalan18102001) [reply Sivabalan/15-02-2023 Task/Main page Script.js:20]: Checked

File: Sivabalan/15-02-2023 Task/Main page Script.js [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+        {
+            var arrObj = [];
+            arrObj.push(myObj);
+            localStorage.setItem("testJSON", JSON.stringify(arrObj));
+            window.open("Next page.html", '_blank');
+        }
+    }
+}
+function resetlocalStorage()
+{
```
[the remark this answers is not in the document]
Comment (sivabalan18102001) [reply Sivabalan/15-02-2023 Task/Main page Script.js:56]: Completed

File: Sivabalan/15-02-2023 Task/Main page Script.js [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+localStorage.clear();
+}
+function userDetails()
+{
+window.open("Next page.html", '_blank');
```
[the remark this answers is not in the document]
Comment (sivabalan18102001) [reply Sivabalan/15-02-2023 Task/Main page Script.js:61]: Completed

File: Sivabalan/15-02-2023 Task/Next page.html [hunk 6]
```diff
@@ -20,80 +19,16 @@
...
-              alert("Enter the Number inside the Table");
-            }
-          }
-          else{
-            alert("Enter the Valid Number in the box");
-          }
-        }
-    </script>
-  </tbody>
+      <script src="Detail page Script.js"></script>
```
[the remark this answers is not in the document]
Comment (sivabalan18102001) [reply Sivabalan/15-02-2023 Task/Next page.html:30]: It write the details inside the table at that place. So script is added in this place.

File: Sivabalan/15-02-2023 Task/User Edit Page.html [hunk 7]
```diff
@@ -69,63 +68,9 @@
...
-            var obj = localStorage.getItem("testJSON");
-            var arr = JSON.parse(obj);
-            arr.splice(No-1,1,myObj);
-            localStorage.setItem("testJSON", JSON.stringify(arr));
-        }
-        function cancelEdit(){
-            window.location.href = "Next page.html";
-        }
-    </script>
+    <script src="Edit page Script.js"></script>
```
[the remark this answers is not in the document]
Comment (sivabalan18102001) [reply Sivabalan/15-02-2023 Task/User Edit Page.html:74]: Completed

File: Sivabalan/15-02-2023 Task/Style for Next second page.css [hunk 8]
```diff
@@ -1,34 +1,59 @@
...
     border-collapse: collapse;
 }
-button{
-    background-color: black;
-    width:max-content;
-    height: 50px;
-    font-size: 30px;
-    border:none;
-    color: white;
+tr:hover{
```
[the remark this answers is not in the document]
Comment (sivabalan18102001) [reply Sivabalan/15-02-2023 Task/Style for Next second page.css:28]: Completed

File: Sivabalan/15-02-2023 Task/Style for Next second page.css [hunk 9]
```diff
@@ -1,34 +1,59 @@
-body{
+body
```
[the remark this answers is not in the document]
Comment (sivabalan18102001) [reply Sivabalan/15-02-2023 Task/Style for Next second page.css:1]: Completed

**Pull request merged**